### PR TITLE
Astro 1284 update image assets for astro uxds

### PR DESCRIPTION
**Pull request by Design-Kat**
https://rocketcom.atlassian.net/browse/ASTRO-1284

Should be all the currently updated Image Assets for PI 2021-1.

@amalyah please review when you get a chance to ensure I've included all the correct images. Not included: Any image from service-specific folders, and images blocked out as still needing update/links to Astro 6.

### CODE REVIEW

**Comment by Design-Kat** [comment]
Note: All images should have been updated that need updating at this point (previously Sign-in was still being completed).

File: _content/components/sign-in.md [hunk 1]
```diff
@@ -16,10 +16,12 @@ theme: true
 The Sign in page is often the first interaction a user has with your product. This entry point experience is an important moment in establishing your product’s brand and experience and sets the tone for their overall experience with the product.
 “Signing in” provides an authentication method to validate a user’s unique identification and allow them access to protected areas of an application. A user provides their required identification and, if correct, they are provided access to the product.
 
- ::: caution 
- When implementing validation and security measures Astro users should consult with their organization's IT/Security departments and adapt these patterns to suit the requirements of their organization’s best practices and security needs. 
+ ::: caution
+ When implementing validation and security measures Astro users should consult with their organization's IT/Security departments and adapt these patterns to suit the requirements of their organization’s best practices and security needs.
  :::
 
+[Example of a full-page, simple sign in screen](/img/components/signin-simple-full.png "Example of a full-page, simple sign in screen")
```
Comment (dmcalester) [inline _content/components/sign-in.md:23]: Please add an exclamation point (!) here to get the image to render e.g. 

```
![Example of a full-page, simple sign in screen](/img/components/signin-simple-full.png "Example of a full-page, simple sign in screen")
```

File: _content/components/sign-in.md [hunk 2]
```diff
@@ -35,34 +37,46 @@ When a user has been signed out due to inactivity. In this scenario, when the us
...
-MFA requires the user to set up additional methods of identification, such as a PIN, or
-4. The Sign in can currently be configured to display Sign in with username or Email plus password, and validation.
+4. Sign in with Multi-Factor Authentication (MFA)
+MFA requires the user to set up additional methods of identification, such as a PIN, or through a verification code sent to a user's mobile device.
+
+The Sign in can currently be configured to display _Sign in with Username or Email Plus Password_, _Sign in with SSO_ and validation.
 
 ### Sign in with Username or Email Plus Password
 
+[Example of a sign in screen hosted within a modal dialog](/img/components/signin-dialog-simple.png "Example of a sign in screen hosted within a modal dialog")
```
Comment (dmcalester) [inline _content/components/sign-in.md:53]: Exclamation point

File: _content/components/sign-in.md [hunk 3]
```diff
@@ -35,34 +37,46 @@ When a user has been signed out due to inactivity. In this scenario, when the us
...
+
 The sign in component consists of a field to identify the user, typically a username or email, and a masked field for a unique passcode, as well as a button that submits the content.  
 
+### Sign in with Single Sign-on (SSO)
+
+[Example of Single Sign-on hosted within a modal dialog](/img/components/signin-dialog-sso.png "Example of Single Sign-on hosted within a modal dialog")
```
Comment (dmcalester) [inline _content/components/sign-in.md:59]: Please add exclamation point before the opening bracket

File: _content/components/sign-in.md [hunk 4]
```diff
@@ -35,34 +37,46 @@ When a user has been signed out due to inactivity. In this scenario, when the us
...
+
+The sign in component consists of a field to identify the user, typically a username or email, and a button that submits the content for SSO verification.  
+
 ### Placement
 
 The sign in UI elements can be added in 3 possible placement areas.
 
+[Example of a sign in screen hosted within a modal dialog](/img/components/signin-dialog-simple.png "Example of a sign in screen hosted within a modal dialog")
```
Comment (dmcalester) [inline _content/components/sign-in.md:67]: Please add exclamation point before the opening bracket

File: _content/components/sign-in.md [hunk 5]
```diff
@@ -35,34 +37,46 @@ When a user has been signed out due to inactivity. In this scenario, when the us
...
 1. Modal dialog
 * Draws user attention to sign in process
 * Can allow application to be running in the background when signed out
 * Ideal for switching applications
 
+[Example of a sign in screen located within a sidebar](/img/components/signin-sidebar-simple-full.png "Example of a sign in screen located within a sidebar")
```
Comment (dmcalester) [inline _content/components/sign-in.md:73]: Please add exclamation point before the opening bracket

File: _content/components/sign-in.md [hunk 6]
```diff
@@ -35,34 +37,46 @@ When a user has been signed out due to inactivity. In this scenario, when the us
...
 2. Sidebar
 * Generally placed on left side of page
 * Allows imagery/branding on opposite side of page to differentiate application
 * Often used alongside SSO
 
+[Example of a full-page sign in screen](/img/components/signin-simple-full.png "Example of a full-page sign in screen")
```
Comment (dmcalester) [inline _content/components/sign-in.md:79]: Please add exclamation point before the opening bracket

File: _content/components/sign-in.md [hunk 7]
```diff
@@ -88,31 +102,33 @@ Common server-side errors:
 * Account lock out
 * Database or system error
 
+[Example of a full-page account lock out (access denied) screen](/img/components/signin-accessdenied-full.png "Example of a full-page account lock out (access denied) screen")
```
Comment (dmcalester) [inline _content/components/sign-in.md:105]: Please add exclamation point before the opening bracket

**Review by dmcalester — CHANGES_REQUESTED** [review]
See inline comments all reference adding a missing exclamation point to some images so they render. Otherwise approved.

**Comment by Design-Kat** [comment]
> See inline comments all reference adding a missing exclamation point to some images so they render. Otherwise approved.

Should be all in now, thank you, @dmcalester !

**Review by amalyah — CHANGES_REQUESTED** [review]
- Design Guidelines/Grid images look blurry. If we add them to the re-export, could that be fixed?
- Gradual Visualization Palette image updates needs to be removes or they need to be re-exported without the NEEDS TO BE LINKED text and opacity cover over them.
- Help text do and dont images aren't showing up on the Forms and Validation page.
- New checkbox images aren't showing up on the checkbox page
- Sign In large images aren't showing up on the Sign In page.
- Some images changed size slightly, others by a lot. Not sure if that's an issue or not.

**Comment by Design-Kat** [comment]
@amalyah 

* Design Guidelines/Grid images look blurry: This is because it’s a bitmap, the original needs to be found and added- maybe that should be a separate ticket/bug? 
*  Gradual Visualization Palette: I re-exported them without the opacity over it.
* Fixed some folder and image name issues on Checkbox, sign in, and forms & validation that should fix the issues you saw with images showing up.

**Review by dmcalester — CHANGES_REQUESTED** [review]
The full screen images for modal, sidebar and full page are mis-rendering @nortonprojects could you lend a hand?

**Review by amalyah — CHANGES_REQUESTED** [review]
Only thing that I see wrong now is the Sign In images that Duncan already mentioned.

**Review by amalyah — APPROVED** [review]
Styling issue: Numbered bullet points do not keep the numbers that Kat entered in the code.

**Review by dmcalester — APPROVED** [review]
Approved for image update.

**Review by amalyah — APPROVED** [review]
Numbering issue is fixed now too.

**Comment by nortonprojects** [comment]
> Numbering issue is fixed now too.

Touching up the sizing, and I'll clean up the git history a bit

**Review by dmcalester — APPROVED** [review]

**Review by cskerbo — APPROVED** [review]

**Review by amalyah — APPROVED** [review]

**Comment by Design-Kat** [comment]
@dmcalester @Cheshire89 it looks like this has all the approvals required as of yesterday, could this be merged so the ticket can be closed?

**Pull request merged**